libs/island-ui/contentful/src/lib/richTextRendering.tsx

```diff
@@ -0,0 +1,174 @@
+import EmbeddedVideo from './EmbeddedVideo/EmbeddedVideo'
+import StaticHtml from './StaticHtml/StaticHtml'
+
+export interface RenderNode {
+  [k: string]: (node: Block | Inline, children: ReactNode) => ReactNode
+}
+
+interface RenderConfig {
+  renderComponent?: (slice: Slice, renderNode: RenderNode) => ReactNode
+  renderWrapper?: (slice: Slice, children: ReactNode) => ReactNode
+  renderPadding?: (top: Slice, bottom: Slice) => ReactNode
+  renderNode?: RenderNode
+  htmlClassName?: string
+}
+
+export const defaultRenderComponent = (
+  slice: Slice,
+  renderNode: RenderNode,
+  htmlClassName?: string,
+): ReactNode => {
+  switch (slice.__typename) {
+    case 'Html':
+      return renderHtml(slice.document, {
+        className: htmlClassName,
+        renderNode,
+      })
+
+    case 'FaqList':
+      return <FaqList {...slice} />
+
+    case 'Statistics':
+      return <Statistics {...slice} />
+
+    case 'Image':
+      return <Image type="apiImage" image={slice} />
+
+    case 'ProcessEntry':
+      return <ProcessEntry {...slice} />
+
+    case 'EmbeddedVideo':
+      return <EmbeddedVideo {...slice} />
+
+    default:
+      return null
+  }
+}
+
+export const defaultRenderWrapper = (
+  _slice: Slice,
+  children: ReactNode,
+): ReactNode => {
+  // TODO: when the grid supports it, we'll create a wrapper here to indent
+  // left and/or right side when needed
+  return children
+}
+
+const typography = (
+  variant: TypographyProps['variant'] & TypographyProps['as'],
+) => (_: Block, children: ReactNode) => (
+  <Typography variant={variant} as={variant}>
+    {['h2', 'h3'].includes(variant) ? (
+      // TODO: stop this data-sidebar-link madness
+      <span data-sidebar-link={String(children)}>{children}</span>
+    ) : (
+      children
+    )}
+  </Typography>
+)
+
+export const defaultRenderNode: Readonly<RenderNode> = {
+  [BLOCKS.HEADING_1]: typography('h1'),
+  [BLOCKS.HEADING_2]: typography('h2'),
+  [BLOCKS.HEADING_3]: typography('h3'),
+  [BLOCKS.HEADING_4]: typography('h4'),
+  [BLOCKS.HEADING_5]: typography('h5'),
+  [BLOCKS.PARAGRAPH]: typography('p'),
+  [BLOCKS.QUOTE]: (_node: Block, children: ReactNode): ReactNode => (
+    <Blockquote>{children}</Blockquote>
+  ),
+  [INLINES.HYPERLINK]: (node: Inline, children: ReactNode): ReactNode => (
+    <Hyperlink href={node.data.uri}>{children}</Hyperlink>
+  ),
+}
+
+export const renderHtml = (
+  document: Document,
+  {
+    renderNode = defaultRenderNode,
+    className,
+  }: {
+    renderNode?: RenderNode
+    className?: string
+  } = {},
+): ReactNode => {
+  return (
+    <StaticHtml className={className}>
+      {documentToReactComponents(document, { renderNode })}
+    </StaticHtml>
+  )
+}
+
+export const defaultRenderPadding = (top: Slice, bottom: Slice): ReactNode => {
+  const [a, b] = [top, bottom].map((slice) => slice.__typename)
+
+  if (a === b) {
+    return <Box paddingTop={3} />
+  }
+
+  if ([a, b].some((s) => s === 'Statistics')) {
```

Comment:
I'm not sure if I understand this correctly, but if there is a "Statistics" slice below this slice, then we want the current slice to have paddingTop = 6?

Reply:
if this is something that will be custom for more slices then it could be better to use a config object for the padding instead of more if statements

Reply:
Yeah, we could make a config here. This is just difficult to get right because it's unclear/unknown if the design will demand different padding between identical components on different pages types (e.g. 20px between A and B on article page, but 100px on landing pages), and the possible combination of components above/below each other is huge.

Ideally I would like to do this with css:
```
.RichText > .Statistics + .ProcessEntry {
    padding-top: ...;
}
```
but believe treat makes that difficult/impossible